app/services/commitments/calculate_amount_service.rb

```diff
@@ -23,27 +23,10 @@ def call
     def commitment_amount_cents
```

Comment:
I am not sure why `invoice_subscription` is instance variable in `Commitments::CalculateAmountService` class. If `Commitments::CalculateAmountService` should calculate true-up based on invoice, I think that we should pass invoice reference to the constructor and make it an instance variable.
Btw, Can you please describe where `Commitments::CalculateAmountService` will be called?

Reply:
`Commitments::CalculateAmountService` needs `invoice_subscription` because it has `Commitments::HelperService` as dependency that needs it to access not only `invoice` but also the `subscription`.

`Commitments::CalculateAmountService` is called [here](https://github.com/getlago/lago-api/blob/feat-min-spend-true-up/app/services/commitments/minimum/calculate_true_up_fee_service.rb#L31)